src/Uno.UI/UI/Xaml/DependencyPropertyCallbackManager.cs

```diff
@@ -21,21 +21,17 @@ internal class DependencyPropertyCallbackManager
 
         public DependencyPropertyCallbackManager()
         {
-            // Suppress the finalizer for this object, re-register when the shadow cache is created.
-            GC.SuppressFinalize(this);
         }
 
-        ~DependencyPropertyCallbackManager()
-        {
-            ReturnShadowToPool();
-        }
+		internal void Dispose()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either implement 'IDisposable.Dispose', or totally rename this method to prevent confusion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)